File: data/gpt-4o/readable_format_HPEC/fortran/file233.f90
subroutine print_array(n, outArray)
implicit none

DATA_TYPE, dimension(n) :: outArray
integer :: n
integer :: i
do i = 1, n
write(0, DATA_PRINTF_MODIFIER) outArray(i)
if (mod(i - 1, 20) == 0) then
write(0, *)
end if
end do
end subroutine

File: data/StarCoder2/readable_format_HPEC/cpp/file233.cpp
#include <iostream>
#include <iomanip> // For std::setw

// Assuming DATA_TYPE is int for this example
void print_array(int n, int* outArray) {
    for (int i = 0; i < n; ++i) {
        std::cout << std::setw(DATA_PRINTF_MODIFIER) << outArray[i]; // Adjust width as needed
        if ((i + 1) % 20 == 0) {
            std::cout << std::endl;
        }
    }
}

// Example usage
int main() {
    const int n = 5;
    int array[n] = {1, 2, 3, 4, 5};
    print_array(n, array);
    return 0;
}
